spec/std/complex_spec.cr

```diff
@@ -197,4 +197,9 @@ describe "Complex" do
     Complex.new(1.2, 0).zero?.should eq false
     Complex.new(1.2, 3.4).zero?.should eq false
   end
+
+  it "rounds" do
+    (Complex.new(1.125, 0.875).round(digits=2)).should eq(Complex.new(1.13, 0.88))
```

Comment:
```suggestion
    (Complex.new(1.125, 0.875).round(2)).should eq(Complex.new(1.13, 0.88))
```

Either `digits: 2` or `2`. Otherwise is a local variable.